packages/components/fields/async-creatable-select-field/src/async-creatable-select-field.stories.tsx

```diff
@@ -0,0 +1,193 @@
+import type { Meta, StoryFn } from '@storybook/react';
+import AsyncCreatableSelectField from './async-creatable-select-field';
+import { useEffect, useState } from 'react';
+import { iconArgType } from '@/storybook-helpers';
+
+const meta: Meta<typeof AsyncCreatableSelectField> = {
+  title: 'Form/Fields/AsyncCreatableSelectField',
+  // @ts-expect-error, @todo: fix typings of field component
+  component: AsyncCreatableSelectField,
+  argTypes: {
+    // @ts-expect-error, @todo: fix typings of field component
+    hintIcon: iconArgType,
+    iconLeft: iconArgType,
+    id: { control: { type: 'text' } },
+    isMulti: { control: { type: 'boolean' } },
+    hint: { control: { type: 'text' } },
+    description: { control: { type: 'text' } },
+    badge: { control: { type: 'text' } },
+    name: { control: { type: 'text' } },
+    touched: { control: { type: 'boolean' } },
+    'aria-label': { control: { type: 'text' } },
+    'aria-labelledby': { control: { type: 'text' } },
+    backspaceRemovesValue: { control: { type: 'boolean' } },
+    containerId: { control: { type: 'text' } },
+    isDisabled: { control: { type: 'boolean' } },
+    placeholder: { control: { type: 'text' } },
+    title: { control: { type: 'text' } },
+    isSearchable: { control: { type: 'boolean' } },
+    isClearable: { control: { type: 'boolean' } },
+    tabIndex: { control: { type: 'text' } },
+    tabSelectsValue: { control: { type: 'boolean' } },
+    cacheOptions: { control: { type: 'boolean' } },
+    allowCreateWhileLoading: { control: { type: 'boolean' } },
+    createOptionPosition: { control: 'select', options: ['first', 'last'] },
+    menuShouldBlockScroll: { control: { type: 'boolean' } },
```

Comment:
Seems like a good thing to do once the field components are all back in one branch again. I created another checkbox in my sb ticket so I won't forget.

I still hope we can consolidate all those `<InsertExtremelyLongNameHereField/>` components into more generic `<Field*>` components (`FieldLabel`, `FieldDescription`, `FieldError`, `FieldInput`, etc..) that can be just wrapped around existing inputs. I created a task for this issue as well, but it will need quite some refinement.